expandybird/test/templates/helpers/common.py

```diff
@@ -1,5 +1,3 @@
-# Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
Do we want to remove these, or put in the standard one that's used throughout the repo?